compiler-rt/lib/dfsan/dfsan.cpp

```diff
@@ -143,21 +163,39 @@ SANITIZER_INTERFACE_ATTRIBUTE uptr __dfsan_shadow_ptr_mask;
 // | reserved by kernel |
 // +--------------------+ 0x0000000000
 
-typedef atomic_dfsan_label dfsan_union_table_t[kNumLabels][kNumLabels];
-
+//typedef atomic_dfsan_label dfsan_union_table_t[kNumLabels][kNumLabels];
+typedef atomic_dfsan_label old_union_table_t[oldNumLabels][oldNumLabels];
```

Comment:
This is going to get removed once we switch to the hash map, right?

Reply:
this is actually not used except during address calculations for the mmaped region. We won't be using it at all, and instead using a hashtable within polytracker (just unordered map, or we can just use the taint forest)